Formula/bitcoin.rb

```diff
@@ -1,16 +1,18 @@
 class Bitcoin < Formula
   desc "Decentralized, peer to peer payment network"
   homepage "https://bitcoin.org/"
-  revision 1
+  revision 2
+
+  SOURCE_CODE_REMOTE_URL = "https://github.com/bitcoin/bitcoin".freeze
 
   stable do
-    url "https://bitcoin.org/bin/bitcoin-core-0.15.1/bitcoin-0.15.1.tar.gz"
+    url "https://bitcoin.org/bin/bitcoin-core-0.16.0/bitcoin-0.16.0.tar.gz"
```

Comment:
I amend `revision` and `url` tags, but not `sha` tags since I forget what is hashed to generate those. Please LMK and I will update the SHA256 tags as well!